src/main/scala/com/datastax/gatling/plugin/model/DseCqlAttributes.scala

```diff
@@ -22,33 +24,37 @@ import io.gatling.core.check.Check
   * @param statement        CQL Statement to be sent to Cluster
   * @param cl               Consistency Level to be used
   * @param cqlChecks        Data-level checks to be run after response is returned
-  * @param genericChecks    Low-level checks to be run after response is returned
-  * @param userOrRole       User or role to be used when proxy auth is enabled
-  * @param readTimeout      Read timeout to be used
   * @param idempotent       Set request to be idempotent i.e. whether it can be applied multiple times
-  * @param defaultTimestamp Set default timestamp on request, overriding current system time
+  * @param node             Set the node that should handle this query
+  * @param customPayload    Custom payload for this request
```

Comment:
Unless I am mistaken, this field is never used.  The only place [where custom payloads are used](https://github.com/absurdfarce/gatling-dse-plugin/blob/803c2c770f789ade51a1e21bf636565b8c8b3d99/src/main/scala/com/datastax/gatling/plugin/model/DseCqlStatements.scala#L226-L242) is when users write something like 

```scala
val stmt = new SimpleStatement(...)
//...
cql("My request").executeCustomPayload(stmt, "sessionKey")
```

There is no code path that leads to writing to this `DseCqlAttributes::customPayload` field, I suggest to remove it entirely.

Reply:
The parallel here is to addCustomPayload() in the old API.  I think I just missed adding it in DseCqlAttributesBuilder, so my inclination is to just add it there in order to make sure the API is complete.  Does that sound right @pingtimeout ?